cache/plan.go

```diff
@@ -87,7 +87,8 @@ func (s *RawProviderServer) PlanResourceChange(ctx context.Context, req *tfproto
 		return resp, nil
 	}
 
-	if proposedVal["timestamp"].IsNull() {
```

Comment:
We now run the "create" logic if its a new resource (timestampe is null) or if the keepers field changes.

The main issue I couldn't figure out is that this can't distinguish between a missing keepers field, and an empty keepers. So doing this:
```
# before
resource "cache_store" "foo" {
  value = "whatever"
}
-------------------------------
# after
resource "cache_store" "foo" {
  value = "whatever"
  keepers = {}
}
```

looks like a "change" and will trigger resetting the cache.